### PR TITLE
Add Documentation for Algo Settings/Builds Tabs

### PR DESCRIPTION
**Pull request by langlk**
Resolves [FRONTEND-1342](https://algorithmia.atlassian.net/browse/FRONTEND-1342)

NOTE: Should wait to merge till after we've deployed https://github.com/algorithmiaio/web/pull/1466

### CODE REVIEW

**Comment by andryanmiller** [comment]
@peckjon @StephanieKim If either of you have time to take a gander at this today, would be appreciated! This feature is expected to ship tomorrow.

**Comment by langlk** [comment]
> Additions LGTM but can you remind me why pricing/costs is being removed? @andryanmiller

We moved all the actual information on that page (cost, avg call duration) to the overview page.

File: _pages/platform/algorithm-profiles.md [hunk 1]
```diff
@@ -61,6 +52,47 @@ If the algorithm developer has published under an Open Source license, you will
...
 
+### Profiles for Your Owned Algorithms
+
+Profiles for algorithms owned by your account, or one of your organizations, will have additional features. You will see two additional tabs: "Settings" and "Builds." Additionally, you will always see the "Source" tab.
+
+![Owned Algorithm Profile]({{site.cdnurl}}{{site.baseurl}}/images/post_images/algorithm_profiles/owned_algo_profile.png)
+
+#### Algorithm Settings
+
+The top of the Settings page includes information for working on an algorithm locally, as well as a tracker for the [algorithm checklist]({{site.baseurl}}/algorithm-development/algorithm-basics/algorithm-checklist). Below these you can edit your algorithm's settings, including source code visibility, license, hardward type, and the other permissions listed [here]({{site.baseurl}}/platform/permissions).
```
Comment (andryanmiller) [inline _pages/platform/algorithm-profiles.md]: 🔠 Minor typo: "hardward".

Comment (langlk) [reply _pages/platform/algorithm-profiles.md]: Thanks for catching that, fixed!

**Pull request merged**